docs/README_JA.md

```
@@ -0,0 +1,180 @@
+- 🕵️ 自律エージェント
+- 📈 ビジネスプロセスの処理
+- 🎮 ビデオゲームのNPC
+
+# 始め方
+
+**必須条件:**
+
+- [Node.js 23+](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+- [pnpm](https://pnpm.io/installation)
+
+### .envファイルの編集
```

Markdownlint (0.37.0)

Expected: h2; Actual: h3
Heading levels should only increment by one level at a time

(MD001, heading-increment)